build_raspOVOS_ca.sh

```
@@ -5,27 +5,25 @@
 # scroll back and figure out what went wrong.
 set -e
 
+## Intended to run on top of the image generated by build_raspOVOS.sh
+
 # Activate the virtual environment
 source /home/$USER/.venvs/ovos/bin/activate
```

Shellcheck (0.10.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)